### PR TITLE
Add a shorter video introduction to the home page

### PR DESCRIPTION
**Pull request by meghanrjones**
**Description of proposed changes**

Following the discussion from https://github.com/GenericMappingTools/pygmt/issues/613, this PR updates the video on the PyGMT landing page to be a shorter intro to PyGMT. I could remake the video if anyone has suggestions that they want included.

Fixes #

**Reminders**

- [ ] Run `make format` and `make check` to make sure the code follows the style guide.
- [ ] Add tests for new features or tests that would have caught the bug that you're fixing.
- [ ] Add new public functions/methods/classes to `doc/api/index.rst`.
- [ ] Write detailed docstrings for all functions/methods.
- [ ] If wrapping a new module, open a 'Wrap new GMT module' issue and submit reasonably-sized PRs.
- [ ] If adding new functionality, add an example to docstrings or tutorials.

**Slash Commands**

You can write slash commands (`/command`) in the first line of a comment to perform
specific operations. Supported slash commands are:

- `/format`: automatically format and lint the code
- `/test-gmt-dev`: run full tests on the latest GMT development version

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -44,14 +44,16 @@ Why PyGMT?
 A beautiful map is worth a thousand words.
 To truly understand how powerful PyGMT is, play with it online on `Binder <https://github.com/GenericMappingTools/try-gmt>`__!
 But if you need some convincing first, watch this `1 hour introduction <https://www.youtube.com/watch?v=SSIGJEe0BIk>`__ to PyGMT!
+For a quicker introduction, check out our `3 minute overview <https://youtu.be/4iPnITXrxVU>`__!
```
Comment (weiji14) [inline README.rst:47]: I think we can just keep the 3 min part here. Suggest moving the 1 hour intro sentence after the 3 min video.

**Comment by weiji14** [comment]
Wow, this is awesome! The video looks really professional, and I think you summarized most of the key features of PyGMT along with the open source community aspect nicely. Definitely something to get on the front page for PyGMT v0.6.0 :smile:

File: README.rst [hunk 2]
```diff
@@ -44,14 +44,16 @@ Why PyGMT?
 A beautiful map is worth a thousand words.
 To truly understand how powerful PyGMT is, play with it online on `Binder <https://github.com/GenericMappingTools/try-gmt>`__!
 But if you need some convincing first, watch this `1 hour introduction <https://www.youtube.com/watch?v=SSIGJEe0BIk>`__ to PyGMT!
```
Comment (seisman) [inline README.rst:46]:
```suggestion
But if you need some convincing first, watch this `1 hour introduction <https://youtu.be/SSIGJEe0BIk>`__ to PyGMT!
```

File: README.rst (hunk 1 above)
Comment (seisman) [reply README.rst:47]: Perhaps we should remove the link to the 1-hour video, because we already have more update-to-date tutorials and gallery examples.

Comment (meghanrjones) [reply README.rst]: I removed the tutorial link from this paragraph and added a reference to the external resources page in the following paragraph. Let me know if you rather only keep references to our internal gallery and examples.

File: README.rst (hunk 2 above)
Comment (meghanrjones) [reply README.rst]: Marking this as resolved since there will no longer be a direct link.

**Review by seisman — APPROVED** [review]
Looks great to me.

**Pull request merged**